rust/core/src/databases.rs

```diff
@@ -134,6 +135,11 @@ pub trait DatabaseQueryGenerator: SnapshotGenerator + Debug + Sync {
 
     /// database_name_wrapper returns a full path or name wrapped in quotes that work for the specific database
     fn database_name_wrapper(&self, name: &str) -> String;
+
```

Comment:
I am not sure about this passing about a string. Feels quite dangerous, maybe we add a type alias to at least make the connection between this and the generate Query function more explicit. 

type Timestamp = String;